data/transition-sites/directgov_los.yml

```
@@ -0,0 +1,6 @@
+---
+site: directgov_los
+whitehall_slug: government-digital-service
+homepage: https://www.gov.uk/government/organisations/government-digital-service
+tna_timestamp: 20181120000000
```

How did you find this timestamp? I think it should be `20180920044754` from looking at https://web.archive.org/web/*/http://los.direct.gov.uk

```suggestion
tna_timestamp: 20180920044754
```

---

I tried, but might have typed it wrong - it often doesn't work and isn't necessary for anything. I'll update it though 👍 .